en/docs/develop/product-apis/gateway-apis/gateway-v1/guide.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started
+
+WSO2 API Manager is a complete solution for publishing APIs, creating and managing a developer community, and for scalably routing API traffic. It leverages proven, production-ready, integration, security and governance components from WSO2 Enterprise Service Bus, WSO2 Identity Server, and WSO2 Governance Registry. Moreover, it is powered by WSO2 Business Activity Monitor, thereby making WSO2 API Manager ready for any large-scale deployments right away.
+
+The REST API is implemented based on REST best practices and specifications as a CXF REST web application running on WSO2 API Manager. API development is started with a Swagger specification with a contract-first approach. Please see the full [Swagger definition](https://raw.githubusercontent.com/wso2/carbon-apimgt/v6.6.163/components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/main/resources/admin-api.yaml), which is written using Swagger 2.0. This can be also retrieved from the Web app itself using the URL `https://<host-name[:port]>/api/am/admin/v0.16/swagger.json`.
```

Comment:
This can be rewritten as follows : - The REST API is implemented based on the REST specification and best practices as a CXF REST web application running on WSO2 API Manager. A contract-first approach could be utilized for API development starting with a Open API specification. See the admin APIs  [Swagger definition](https://raw.githubusercontent.com/wso2/carbon-apimgt/v6.6.163/components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/main/resources/admin-api.yaml), which is written using Open API 3.0 .Alternatively, this API definition can be accessed using the product URL as well `https://<host-name[:port]>/api/am/admin/v0.16/swagger.json`.
Note to use the correct OPen API definition URL and the product API version URL for the latest product version

Reply:
https://github.com/wso2/docs-apim/pull/1527/commits/46b28b730c7fd76e644b5b021cac43b057a7b265